libpod/container_internal_linux.go

```diff
@@ -2367,3 +2378,37 @@ func (c *Container) checkFileExistsInRootfs(file string) (bool, error) {
 	}
 	return true, nil
 }
+
+// getSecret gets a secret to mount into the container
+func (c *Container) getSecret(name string) (string, string, error) {
+	secretsDir := filepath.Join(c.runtime.store.GraphRoot(), "secrets")
```

Comment:
Make an accessor for this somewhere - we're going to need a way to determine where these live.